src/remove_plugin.py

```diff
@@ -1,7 +1,98 @@
+import logging
+import time
 import click
+from atlassian.jira import JiraServer
+from config import Config
+from utils.apprise_notify import NotifyType, apprise_notify
 
+logger = logging.getLogger(__name__)
 
-@click.command(help="Remove plugin on instance")
-def remove_plugin_server():
-    click.echo("Removing plugin")
-    click.echo("NOT IMPLEMENTED")
+
+@click.command(help="Remove plugin on Atlassian server instance")
+@click.argument("plugin_key", type=str)
+@click.option("--url", "-url", default="", type=str, help="Atlassian instance base URL")
+@click.option("--username", "-u", default="", type=str, help="Username to login")
+@click.option("--password", "-p", default="", type=str, help="Password to login")
+@click.option(
+    "--notify",
+    "-n",
+    default=[],
+    type=str,
+    multiple=True,
+    help="Apprise notification URL",
+)
+@click.option("--timeout", default=120, help="Remove timeout in second")
+def remove_plugin_server(
+    plugin_key: str,
+    url: str,
+    username: str,
+    password: str,
+    notify: list[str],
+    timeout: float,
+):
+    vars = {
+        "url": url if url else Config.ATLAS_URL,
+        "username": username if username else Config.ATLAS_USERNAME,
+        "password": password if password else Config.ATLAS_PASSWORD,
+    }
+    for key, val in vars.items():
+        if val == "":
+            raise ValueError(f"'{key}' is not set")
+
+    if len(notify) == 0 and Config.NOTIFY_TITLE != "":
```

Comment:
it should be `Config.NOTIFY_URL`, I fixed it in https://github.com/itdevsamurai/atlassian-plugin-remote-installation-docker/commit/6f8c458898411a0f6f08302583143a507e1b4eee